app/views/EORIBeUpToDateView.scala.html

```diff
@@ -14,14 +14,57 @@
  * limitations under the License.
  *@
 
+@import components._
+@import config.FrontendAppConfig
+
 @this(
         layout: templates.Layout,
-        govukButton: GovukButton
+        govukButton: GovukButton,
+        formHelper: FormWithCSRF,
+        caption: Caption,
+        heading: Heading,
+        paragraph: Paragraph,
+        link: Link,
+        bulletList: BulletList,
+        cancelApplicationLink: CancelApplicationLink,
+        config: FrontendAppConfig
 )
 
 @()(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm(messages("eORIBeUpToDate.title"))) {
+@eoriBeUpToDateLinkOne = {
```

Comment:
Very minor and theres only two here but we could be more accurate with variable names i.e enquiryFormLink, enquiriesInfoLink